guides/common/assembly_building_cloud-images.adoc

```diff
@@ -10,8 +10,10 @@ include::modules/proc_registering-a-host.adoc[leveloffset=+1]
 
 include::modules/proc_installing-and-configuring-puppet-agent-manually.adoc[leveloffset=+1]
 
+ifdef::satellite[]
```

Comment:
I'm not sure. Aren't they still relevant for upstream?

Reply:
Technically yes, upstream users could still use RHEL 6 (???) or RHEL 7/Oracle Linux 7 with paid support. Though I'd argue that it's OK to drop this by now.

line 13 was basically a judgement call by me. I am happy to adjust this to "if not orcharhino".